src/main/java/emu/grasscutter/game/world/World.java

```diff
@@ -545,12 +545,13 @@ public void updateTime() {
      * @param locked True if the world time should be locked.
      */
     public void lockTime(boolean locked) {
-        this.timeLocked = locked;
-
+        if(host.getProperty(PlayerProperty.PROP_IS_GAME_TIME_LOCKED) != 0) {
+            this.timeLocked = true;
+        }else{
+            this.timeLocked = locked;
+        }
```

Comment:
run `gradlew spotlessApply` or
```suggestion
        if (host.getProperty(PlayerProperty.PROP_IS_GAME_TIME_LOCKED) != 0) {
            this.timeLocked = true;
        } else {
            this.timeLocked = locked;
        }
```

Reply:
Ah! So that's where spotlessApply is!